src/components/EventCard/EventCard.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 
-const EventCard = ({ id, name, image, city, state, date }) => {
+const EventCard = ({ id, name, image, city, state, date, pathname }) => {
   return(
-    <Link to={`/event/${id}`}>
+    <Link to={pathname === '/' ? `/event/${id}` : `/profile/event/${id}`}>
```

Comment:
Nice thinking here! The inline conditional is clean and easy to read. I think this is a good solution to the bug we were experiencing.